### DISCUSSION TITLE
Curve Line Tool

### DISCUSSION THREAD

**Discussion by jasoncmcg** [Orama-Interactive/Pixelorama#1020] [Ideas]
A curve line tool as seen here: https://www.aseprite.org/docs/drawing/
It draws a line and then goes into curve mode where you can set the curve points.
This is very handy for specific smooth shapes like eyes.
Looks like this was on a list about 4 years ago. 
Any thoughts on bringing it back up?

**Comment by Variable-ind** [comment]
Hy 👋, a krita inspired curve tool has been added to pixelorama and is also available as a plugin for 0.11.4 available for download through [VariableStore](https://variable-interactive.itch.io/pixelorama-extensions).
Though if you wish to use the extension, I plan to release a new version of it either today or tomorrow so keep an eye out for it (updates are also done through [VariableStore](https://variable-interactive.itch.io/pixelorama-extensions))

https://github.com/Orama-Interactive/Pixelorama/assets/77773850/164f63d2-903b-4eb7-9f8d-3826767f268c

**Reply by jasoncmcg** [reply]
Oh, that looks awesome! I will see if I can get it installed! Thank you

**Comment by OverloadedOrama** [comment]
This has been recently implemented for version 1.0, which will be coming soon, in #1019.
